mne/viz/utils.py

```diff
@@ -67,6 +67,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
 
     try:  # see https://github.com/matplotlib/matplotlib/issues/2654
         fig.canvas.draw()
+	fig.tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad)
         fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
```

Comment:
@Eric89GXL I fear it's not clear to @fraimondo how you prefer it to be. Feel free to take over if you have a clear vision  of how to optimize it.